configs/components/jsbach/jsbach.datasets.yaml

```diff
@@ -1,7 +1,10 @@
+hist_fold: hist_harvest
+hist_scenario: ssp370  # dataset to use for year > 2014
+
 input_sources:
         lctlib: "${echam.model_dir}/lctlib_nlct21.def"
         "jsbach_1850": "${adj_input_dir}/jsbach_${echam.resolution}${echam.ocean_resolution}_11tiles_5layers_1850${jsbachnc_file_ending}.nc"
-        "jsbach_1976": "${adj_input_dir}/jsbach_${echam.resolution}${echam.ocean_resolution}_11tiles_5layers_1976${jsbachnc_file_ending}.nc"
+        "jsbach_1976": "${adj_input_dir}/jsbach_${echam.resolution}${echam.ocean_resolution}_11tiles_5layers_1850${jsbachnc_file_ending}.nc"
```

Comment:
Why was this reverted back to original? The latest change is from @seb-wahl : https://github.com/esm-tools/esm_tools/commit/db283c26a0a70920b9b932425e0a608c0644901f

It kind of make sense that the 1976 file has for name 1976 and not 1850

Reply:
I didnt do that.

Reply:
I discovered how that happened. On Feb 4 when the `origin/ssp585_awicm-recom` was merged into `feat/ssp585_awicm-recom_v6`, that line was overwritten. It basically took the previous thing at overwrote the new one. I am now reverting it to Sebastian's version